store.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+</head>
+
+<body>
+    <header>
+        <div id="header">
+            <div class="logo">
+                <img src="favicon.ico" alt="logo Karma" />
+            </div>
+         </div>   
+<nav>
+    <div id="nav-bar">
+        <a class="nav-item" href="index.html"><strong>Meet Karma</strong></a>
+        <a class="nav-item" href="#">How it works</a>
+        <a class="nav-item" href="store.html">Store</a>
+        <a class="nav-item" href="#">Blog</a>
+        <a class="nav-item" href="#">Help</a>
+        <a class="nav-item" href="#">Login</a>
+    </div>
+</nav>
+    </header>
+
+
+<main class="main_store">
```

Comment:
Good job for using `header`, `main` and `footer` as top-level tags in this page.